rfcs/007 - Peer review infra changes.md

```diff
@@ -31,23 +31,23 @@ Eng teams move over from using cloud UIs to IaC because cloud infrastructure cha
-- works with `iasql_upgrade`
-- works with any source control solution without the need for special access to it
+- works with IaSQL schema upgrades
+- works with any version control system, not just GitHub, without the need for privileged access
 - preserves the ability to `iasql_preview` changes in development and get the type-safety feedback from the dashboard editor
 
-The workflow simply suggests users run `iasql_begin`, perform all the necessary changes, and before doing `iasql_commit` they manually put the output of `iasql_preview` and the SQL generated thus far into a markdown file within a repository of their liking. Once the PR is approved by their peers and landed, they can manually run `iasql_commit`. We can require a parameter for `iasql_commit(message)` which is a string attached as metadata for the transaction that can be set to be the PR URL. Best practices are thus enforced using convention and not configuration with a complete reference of what was done via the IaSQL audit log if necessary.
+The workflow simply suggests users run `iasql_begin`, perform all the necessary changes, and before doing `iasql_commit` they put the output of `iasql_preview` and the SQL generated thus far into a file within a repository of their liking. Once the PR is approved by their peers and landed, they can manually run `iasql_commit`. We can require a parameter for `iasql_commit(message)` which is a string attached as metadata for the transaction that can be set to be the PR URL. Best practices are thus enforced using convention and not configuration with a complete reference of what was done via the IaSQL audit log if necessary.
 
-The required work in the IaSQL engine is associating entries in the audit log with specific transactions and adding a `get_sql_for_transaction()` that returns the SQL for the current transaction so users don't have to provide a timestamp for `get_sql_since()`. Additionally, we can modify transactions so they are active past 30 minutes and do not expire at a given point in time as users are making changes to the database.
+The required work in the IaSQL engine is associating entries in the audit log with specific transactions and adding a `get_sql_for_transaction()` that returns the SQL for the current transaction so users don't have to provide a timestamp for `get_sql_since()`. To formalize the convention, it would be pretty easy to do a pure pl/pgsql function `iasql_create_review(title text, description text)` function that internally creates the context of the markdown file consisting of the title and description of the change, followed by `sql` code and <table> formatted blocks powered by get_sql_for_transaction() and iasql_preview() respectively. Then they just have to copy-paste this into GitHub or whatever version control system their team uses. The documentation can describe what it's doing under the hood with these two functions but reduce the friction involved in formatting the output for consumption.
```

Comment:
```suggestion
The required work in the IaSQL engine is associating entries in the audit log with specific transactions and adding a `get_sql_for_transaction()` that returns the SQL for the current transaction so users don't have to provide a timestamp for `get_sql_since()`. To formalize the convention, it would be pretty easy to do a pure pl/pgsql function `iasql_create_review(title text, description text)` function that internally creates the context of the markdown file consisting of the title and description of the change, followed by ` ```sql` code and `<table>` formatted blocks powered by get_sql_for_transaction() and iasql_preview() respectively. Then they just have to copy-paste this into GitHub or whatever version control system their team uses. The documentation can describe what it's doing under the hood with these two functions but reduce the friction involved in formatting the output for consumption.
```

Reply:
the markdown file itself did not like the triple quotes but will wrap `<table>`

Reply:
But I left my comment that exact way? You can wrap backticks with backticks using a very precise spacing mechanism

Reply:
Updated the suggestion